R/bmi.R

```diff
@@ -0,0 +1,83 @@
+#' @title ifelse2
+#' 
+#' @description Custom ifelse function that evaluates missing (NA) values
+#' 
+#' @details If the logical argument compares to a value that is NA, it will set `x` to `FALSE`  
```

Comment:
@details If the logical argument (x) compares to a value that is NA, it is set to `FALSE`